examples/seed/README.md

```diff
@@ -0,0 +1,5 @@
+# Trunk | Seed
+
+An example application demonstrating building a WASM web application using Trunk & Seed taken from: [Seed TodoMVC Example](https://github.com/seed-rs/seed/tree/master/examples/todomvc)
+
+Once you've installed Trunk, simply execute `trunk serve --open` from this example's directory, and you should see the web application rendered in your browser.
```

Comment:
I dig it! Do you mind taking a screenshot of the app rendered in the browser (just the app window, no bookmarks toolbars and such 😆 ) and then add that screenshot to the readme here? You can just commit the PNG to the repo directly under this examples/seed/ dir.

Thoughts?

Reply:
I just updated the README.md and added a PNG screenshot.